src/async_impl/client.rs

```diff
@@ -566,8 +569,19 @@ impl ClientBuilder {
                     }
 
                     // Build TLS config
+                    #[cfg(feature = "__rustls_crypto_ring")]
+                    let provider = rustls::crypto::ring::default_provider();
+
+                    #[cfg(all(
+                        feature = "__rustls_crypto_aws_lc-rs",
```

Comment:
```suggestion
                        feature = "__rustls_crypto_aws_lc",
```
feature name is wrong here too